### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by live1206**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/7012)

### CODE REVIEW

**Review by yufeih — COMMENTED** [review]
Need to update this logic to:
- Call `GetHttpConfig` (which internally does sorting, so longest url match)

image

File: src/docfx/lib/git/GitUtility.cs [hunk 1]
```diff
@@ -237,28 +237,31 @@ private static string Execute(string cwd, string commandLineArgs, bool stdout =
...
+                return ProcessUtility.Execute("git", commandLineArgs, cwd, stdout, secret);
             }
             catch (Win32Exception ex) when (ProcessUtility.IsExeNotFoundException(ex))
             {
                 throw Errors.System.GitNotFound().ToException(ex);
             }
         }
 
-        private static (string cmd, string[] secrets) GetGitCommandLineConfig(string url, PreloadConfig config)
+        private static (string cmd, string? secret) GetGitCommandLineConfig(string url, PreloadConfig config)
```
Comment (yufeih) [inline src/docfx/lib/git/GitUtility.cs:248]: `return default` produces `(null, null)`
```suggestion
        private static (string? cmd, string? secret) GetGitCommandLineConfig(string url, PreloadConfig config)
```

**Review by yufeih — APPROVED** [review]

File: src/docfx/lib/git/GitUtility.cs (hunk 1 above)
Comment (live1206) [reply src/docfx/lib/git/GitUtility.cs]: Updated, L252 already broke it

Comment (yufeih) [reply src/docfx/lib/git/GitUtility.cs]: Yep. Better late than never.

**Pull request merged**